cluster/mesos/docker/util.sh

```diff
@@ -61,7 +61,7 @@ function cluster::mesos::docker::docker_compose {
 # to `docker pull` which always hits the remote docker repo, even if the image
 # is already cached.
 function cluster::mesos::docker::docker_compose_lazy_pull {
-  for img in $(grep '^\s*image:\s' "${provider_root}/docker-compose.yml" | sed 's/[ \t]*image:[ \t]*//'); do
+  for img in $(grep '^\s*image:\s' "${provider_root}/docker-compose.yml" | sed 's/[ \t]*image:[ \t]*//' | egrep -v "^local/"); do
```

Comment:
I'm not a big fan of this exception. I'd rather just push the image to dockerhub.